python/python/tests/test_async_client.py

```diff
@@ -7062,7 +7065,14 @@ async def test_function_load(self, redis_client: TGlideClient):
             await redis_client.fcall_ro(func_name, arguments=["one", "two"]) == b"one"
         )
 
-        # TODO: add FUNCTION LIST once implemented
+        # verify with FUNCTION LIST
+        check_function_list_response(
+            await redis_client.function_list(lib_name, with_code=True),
+            lib_name.encode(),
```

Comment:
I hope you can move all `encode` calls inside `check_function_list_response` or even use `decode` there